src/devices/video/315_5124.cpp

```diff
@@ -68,6 +68,25 @@ PAL frame timing
 #include "video/315_5124.h"
 
 
+/****************************************************************************
+ * Configurable logging
+ ****************************************************************************/
+
+#define LOG_GENERAL    (1U <<  0)
+#define LOG_VIDMODE    (1U <<  1)
+#define LOG_REGWRITE   (1U <<  2)
+#define LOG_VCOUNTREAD (1U <<  3)
+
+//#define VERBOSE (LOG_GENERAL | LOG_VIDMODE | LOG_REGWRITE | LOG_VCOUNTREAD)
+//#define LOG_OUTPUT_FUNC printf
```

Comment:
If you’re going to supply something here as an example, it’s better to use an `osd_printf_*` function, because they have the same semantics as `logerror` while `printf` has different semantics depending on the C runtime library.